map/src/map/layers/RouteLayer.js

```diff
@@ -5,7 +5,6 @@ import AppContext from "../../context/AppContext";
 import {useNavigate, useLocation} from 'react-router-dom';
 import MarkerOptions from "../markers/MarkerOptions";
 
-
 function dist(a1, a2) {
```

Comment:
RouteLayer.js must has only functions for working with map

Reply:
> RouteLayer.js must has only functions for working with map

Give me the details: which function you refer to ?

Reply:
closed